definitions.yaml

```diff
@@ -29,6 +29,68 @@ definitions:
           datacenter:
             description: Identifier of the datacenter or cloud provider region, e. g. "eu-west-1"
             type: string
+          availability_zones:
+            type: object
+            description: Number of availability zones that the cluster can be spread across.
+            properties:
+              max:
+                description: Maximum number of availability zones in this region.
+                type: number
+              default:
+                description: Default number of availability zones for a cluster.
+                type: number
+      network:
+        description: Information related to networking in the clusters
+        type: object
+        properties:
+          ranges:
+            description: IP ranges that can be used by clusters
+            type: array
+            items:
+              type: object
+              properties:
+                name:
+                  description: Name of the IP range (eg production or default)
+                  type: string
+                pods:
```

Comment:
Any reason for this order? `clusters`, `docker`, `pods`, `services` would be alphabetical. Am I missing something?

Reply:
I don't know. Any reason to sort it alphabetical?

From a scope point of view its probably:

* clusters
* services
* pods
* docker

Cluster API also specifies services and then pods: https://github.com/kubernetes-sigs/cluster-api/blob/master/pkg/apis/cluster/v1alpha1/cluster_types.go#L58

In fact. I don't care! :smile:

Reply:
Well that would make sense to me as well, currently it is clusters last though - which is why I wondered this :D